Source/mpqapi.cpp

```diff
@@ -313,18 +323,28 @@ BOOL OpenMPQ(const char *pszArchive, DWORD dwChar)
 {
 	DWORD dwFlagsAndAttributes;
 	DWORD key;
-	DWORD dwTemp;
 	_FILEHEADER fhdr;
 
 	InitHash();
 	dwFlagsAndAttributes = gbMaxPlayers > 1 ? FILE_FLAG_WRITE_THROUGH : 0;

```

Comment:
Interesting, caching was disabled for multiplayer. It doesn't look like you kept this behaviour, was that intentional?

Reply:
I didn't notice this! Not sure why it does this.

Should we preserve this? We can it do it by calling `flush` after every write.

Reply:
Looks like we didn't implement it in `CreateFile` previously either, but yeah calling `flush` sounds like the right thing to do. I guess it's just there to prevent people from getting to an earlier save, so it's also not that important really.